stack/stack.go

```diff
@@ -66,7 +66,6 @@ func ParseYAMLData(fileData []byte, regex string, filter string) (*Services, err
 			if regexExists {
 				match, err = regexp.MatchString(regex, function.Name)
 				if err != nil {
-					fmt.Printf("Invalid regex passed to --regex option\n")
```

Comment:
This message was deleted?

Reply:
There is already an error thrown by the regex parser when the regex is invalid, which gets printed out by each command which calls `ParseYAMLFile()` by its following error checking block:

```
if err != nil {
    log.Fataln(err.Error())
    return
}
```

The line that `log.Fataln(err)` prints is something like 'invalid regexp', so I figured this printf is redundant. Also, for testing, I wanted to confirm the error message, and only have one line print out makes that easier.

Reply:
As long as you bubble up the error.